### ISSUE TITLE
Downloading Visualization Data to CSV - Only works in first the click

### ISSUE THREAD

**Issue by ilidio** [alexandersimoes/d3plus#613]
The download only works for the first time you click
http://d3plus.org/examples/advanced/98576e5a455dccd421f3/

**Comment by davelandry** [comment]
@ilidio is this related to #578?

**Comment by ilidio** [comment]
@davelandry No. I think this directly related to the engine that renders data and the chart.

There a strange behavior about that. If you use resize option, If you resize your browser window, you can download it again.

`  var visualization = d3plus.viz()
      ...
	.resize(true)
    .draw()`

**Comment by davelandry** [comment]
CSV downloads removed in 2.0
